Exercise02/Program.cs

```diff
@@ -14,15 +14,24 @@ namespace Exercise02
 {
     class Program
     {
-        public static object obj { set; get; }
-
         static void Main(string[] args)
         {
-            RestAdapter adapter = new RestAdapter(@"https://jsonplaceholder.typicode.com/");
-            IUserService service = adapter.Create<IUserService>();
-            RestResponse<List<User>> userResponse = service.GetUser();
-            List<User> listUser = userResponse.Data;
-            listUser.ForEach(Console.WriteLine);
+            var adapter = new RestAdapter("https://jsonplaceholder.typicode.com/");

```

Comment:
Split this into many segments (by adding empty lines) or extract to many functions and call.